packages/operation/src/evm/utils.ts

```diff
@@ -25,6 +25,21 @@ export function getDesignatedWallet(mnemonic: string, requester: string, provide
   return deriveWalletFromPath(mnemonic, `m/0/${addressToDerivationPath(requester)}`, provider);
 }
 
+/**
+ * HD wallets allow us to create multiple accounts from a single mnemonic.
+ * Each requester creates a designated wallet for each provider to use
+ * in order for them to be able to respond to the requests their clients make.
+ *
+ * By convention derivation paths start with a master index
+ * followed by child indexes that can be any integer up to 2^31.
+ *
+ * Since addresses can be represented as 160bits (20bytes) we can then
+ * split it in chunks of 31bits and create a path in with the following pattern:
+ * m/0/1st31bits/2nd31bits/3rd31bits/4th31bits/5th31bits/6th31bits.
+ *
+ * @param address A string representing a 20bytes hex address
```

Comment:
For what it's worth, we haven't really been using JSDoc comment syntax anywhere other than the Solidity contracts. I don't mind it though

Reply:
Right, I saw that too but I'm thinking that this function might be refactored into it's own package or some common place because it's being used in at least 2 or 3 different packages at the moment.